src/utils/plot-templates/dark.js

```diff
@@ -84,6 +84,7 @@ const darkTemplate = {
     family: 'Titillium+Web:400',
     color: 'rgba(255,255,255,0.55)',
   },
+  height: undefined,
```

Comment:
yes, null works as well. It was null before, then i changed it. This is just because sometimes the plotly data in the api provides height and width values. and i want to override it with null and let the height be specified by the css so that the chart looks expanded